google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/AlarmFactory.java

```diff
@@ -47,4 +47,10 @@ static AlarmFactory create(Duration duration) {
                 duration.toNanos(),
                 NANOSECONDS);
   }
+
+  static AlarmFactory createUnbounded(Duration duration) {
```

Comment:
Can you add a comment stating that the runnable is executed by an unbounded pool, although the alarm pool is bounded.